backend/pokemon/models.py

```diff
@@ -8,3 +8,4 @@ class Pokemon(models.Model):
     attack = models.IntegerField()
     defense = models.IntegerField()
     hp = models.IntegerField()
+    img_url = models.CharField(max_length=100, blank=True, )
```

Comment:
max_length=100 is quite low for an URL. I suggest changing to 255

Reply:
okay